src/hotspot/cpu/riscv/stubGenerator_riscv.cpp

```diff
@@ -5358,8 +5358,11 @@ class StubGenerator: public StubCodeGenerator {
     __ vor_vv(outputV1, outputV1, outputV2);
     __ vmseq_vi(v0, outputV1, -1);
     __ vfirst_m(failedIdx, v0);
-    Label NoFailure;
-    __ beq(failedIdx, minusOne, NoFailure);
+    Label NoFailure, FailureAt0Idx;
```

Comment:
Nit: Maybe rename `FailureAt0Idx` to `FailureAtIdx0`?

Reply:
Yeh, it makes sense!